to_do/to-do-app.js

```diff
@@ -0,0 +1,267 @@
+window.addEventListener("DOMContentLoaded", init);
+
+//Initialization which calls all the other functions
+function init() {
+  const toDoContainer = document.getElementById("list-container");
+  const addButtonNotStarted = document.querySelector(
+    ".new-task-button-not-started"
+  );
+  const addButtonInProgress = document.querySelector(
+    ".new-task-button-in-progress"
+  );
+
+  addButtonNotStarted.addEventListener("click", () =>
+    addTask(toDoContainer, false, 1)
+  );
+  console.log("clicked addbutton not started");
+  addButtonInProgress.addEventListener("click", () =>
+    addTask(toDoContainer, false, 2)
+  );
+  console.log("clicked addbutton in progress");
+
+  // Add saved tasks for Not Started and In Progress from localStorage
+  addTask(toDoContainer, true);
+
+  updateCount();
+}
+
+function addTask(toDoContainer, existing, category) {

```

Comment:
I would add a bit more comments throughout the doc to tell the reader what the code is doing. It's self-explanatory after reading through a few times but having comments before each "group" of tasks would be very helpful!